### PR TITLE
Update Local Setup Tutorial in v2.1.4 Docs

### PR DESCRIPTION
**Pull request by auginator**
Updating the tutorial's instructions for local setup to create a starter environment based on the [codesandbox](https://codesandbox.io/s/formik-v2-tutorial-start-s04yr) used for the rest of the tutorial.  
- Changed links to indicate source should be copied from codesandbox
- Also added npm/yarn instructions for adding all dependencies used in tutorial (`formik yup @emotion/core @emotion/styled`).

-----
[View rendered docs/tutorial.md](https://github.com/auginator/formik/blob/fix-docs-local-tutorial/docs/tutorial.md)

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -68,19 +68,21 @@ del *
...
 
-6. Add these three lines to the top of `index.js` in the `src/` folder:
+6. Add all dependencies used in tutorial with NPM/yarn:
 
-```js
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
+```bash
+npm install --save formik yup @emotion/core @emotion/styled
```
Comment (jaredpalmer) [inline docs/tutorial.md]: just download the sandbox and npm install?

Comment (auginator) [reply docs/tutorial.md]: Got it, thanks Jared. That was a derp on my part – I couldn't find the download functionality on codesandbox.

PR updated. Makes the instructions MUCH shorter!

**Review by maddhruv — APPROVED** [review]
LGTM 🎉

**Comment by jaredpalmer** [comment]
Please also change in the root tutorial not just 2.1.4

**Pull request closed without merging**